doc/releases.md

```diff
@@ -81,6 +81,19 @@ Notes:
 - Version strings are listed below as _"vx.y.z"_. Substitute for the release
   version.
 
+### 0. Pre-release steps
+
+Before preparing a Node.js release, you must notify the Build Working Group at
```

Comment:
I'd /s/must/should.

Also drop a line about "how", i.e. open an issue in the build repo, and check for responses.

Reply:
Honestly, I'd rather keep it must. Good idea to add the "how"

Reply:
I think we try to avoid using “you”? So maybe just delete “you must” together and it means basically the same